src/core/exception.d

```diff
@@ -19,6 +19,31 @@ void __switch_errorT()(string file = __FILE__, size_t line = __LINE__) @trusted
         assert(0, "No appropriate switch clause found");
 }
 
+
+/**
+ * Thrown on a configuration error.
+ */
+class ConfigurationError : Error
```

Comment:
I'm not sure `ConfigurationError` and `ForkError` have to go into this generic module. For example, `ThreadErrror` is defined locally in `core.thread`. Do you expect someone catching this specific type so a definition has to be public?

Reply:
The problem is that throw staticError is private